### DISCUSSION TITLE
How to create a Git Branch

### DISCUSSION THREAD

**Discussion by AbidHasanPiash** [STRINGLABITSOLUTIONS/Bhojon#180] [Q&A] [answered]
Git Branch

**Comment by montasim** [comment, marked as answer]
git checkout -b newBranchName

**Comment by AbidHasanPiash** [comment]
Good

**Comment by Preetiraj3697** [comment]
git branch 'give you branch name'
git checkout branch
git add.
git commit -m"add branch"
git push origin main/master
